src/safe-nonce/safe-nonce.js

```diff
@@ -0,0 +1,16 @@
+htmx.defineExtension('safe-nonce', {
+  transformResponse: function(text, xhr, elt) {
+    if (htmx.config.inlineScriptNonce) {
+      htmx.config.inlineScriptNonce = ''
```

Comment:
have to disable the older script nonce setting so it does not override the nonces on all scripts bypassing the added security